test/e2e/set/scylladbmonitoring/scylladbmonitoring.go

```diff
@@ -132,7 +132,8 @@ var _ = g.Describe("ScyllaDBMonitoring", func() {
 		return fmt.Sprintf("with %q monitoring type", e.Type)
 	}
 
-	g.DescribeTable("should setup monitoring stack", func(e *entry) {
+	// Disabled on OpenShift because of https://github.com/scylladb/scylla-operator/issues/2319#issuecomment-2643287819
+	g.DescribeTable("should setup monitoring stack TESTCASE_DISABLED_ON_OPENSHIFT", func(e *entry) {
```

Comment:
Would it be possible to extend these labels and skip based on it?

https://github.com/scylladb/scylla-operator/blob/e0c2125e99fbd5f97ccbb1f3ecf7484a987c499c/test/e2e/framework/meta.go#L17-L26

Reply:
discussed offline